src/backend/joanie/tests/core/test_models_enrollment.py

```diff
@@ -28,19 +28,37 @@
 class EnrollmentModelsTestCase(TestCase):
     """Test suite for the Enrollment model."""
 
+    def setUp(self):
+        super().setUp()
+        self.now = timezone.now()
+
+    def create_opened_course_run(self, count=1, **kwargs):
+        """Create course runs opened for enrollment."""
+        if count > 1:
+            return factories.CourseRunFactory.create_batch(
+                count,
+                start=self.now - timedelta(hours=1),
+                end=self.now + timedelta(hours=2),
+                enrollment_end=self.now + timedelta(hours=1),
+                **kwargs,
+            )
+
+        return factories.CourseRunFactory(
+            start=self.now - timedelta(hours=1),
+            end=self.now + timedelta(hours=2),
+            enrollment_end=self.now + timedelta(hours=1),
+            **kwargs,
+        )
+
     @mock.patch.object(OpenEdXLMSBackend, "set_enrollment", return_value=True)
     def test_models_enrollment_str_active(self, _mock_set):
         """The string representation should work as expected for an active enrollment."""
         resource_link = (
             "http://openedx.test/courses/course-v1:edx+000001+Demo_Course/course"
         )
 
-        course_run = factories.CourseRunFactory(
-            title="my run",
-            resource_link=resource_link,
-            start=timezone.now() - timedelta(hours=1),
-            end=timezone.now() + timedelta(hours=2),
-            enrollment_end=timezone.now() + timedelta(hours=1),
+        course_run = self.create_opened_course_run(
```

Comment:
I find it harder to read than repeating the call to the factory everywhere.

Reply:
Ok 🤷‍♂️